services/m3dbnode/config/config.go

```diff
@@ -71,6 +71,14 @@ type Configuration struct {
 	// Write new series backoff between batches of new series insertions.
 	WriteNewSeriesBackoffDuration time.Duration `yaml:"writeNewSeriesBackoffDuration"`
 
+	// Tick per series sleep at the completion of a tick batch, only used
+	// if the value is greater than zero otherwise the default is used.
+	TickPerSeriesSleepDuration time.Duration `yaml:"tickPerSeriesSleepDuration"`
+
+	// Tick minimum interval controls the minimum tick interval for the node,
```

Comment:
Since these are "public" config I wonder if its worth making these as explicit as the other comments? Presumably this is what people will look at when configuring their nodes

Reply:
You mean just more descriptive?  These should not need to be overridden TBH, in fact I'm kind of tempted to make them only override-able using config KV values.